packages/paste-core/components/modal/src/Modal.tsx

```diff
@@ -18,6 +18,7 @@ const ModalDialogOverlay = styled(ModalDialogPrimitiveOverlay)(
     alignItems: 'center',
     width: '100%',
     backgroundColor: 'colorBackgroundOverlay',
+    zIndex: '2000', // current zIndex tokens only go up to 90
```

Comment:
Can you add a comment just about this line to make it clear it’s there for console navigation reasons, so no one removes it accidentally.

Thanks

Reply:
Yep. Comment added.